Cython/Utils.py

```diff
@@ -326,9 +343,14 @@ def find_versioned_file(directory, filename, suffix,
         versions = _parse_file_version(path)
         if versions:
             int_version = int(versions[0])
-            # Let's assume no duplicates.
-            if best_match[0] < int_version <= _current_version:
+            if suffix == 'pxd':
+                # Let's assume no duplicates.
+                if best_match[0] < int_version <= _current_version:
+                    best_match = (int_version, path)
+            else:
+                # We pick first .so file available
                 best_match = (int_version, path)
```

Comment:
Here I am not managing python version because I am not 100% sure what should I do with it. Match it with python version under which Cython is running - not sure if it is right to do...

Reply:
We don't care about the Python version. This function is about Cython versioning.